activities.go

```diff
@@ -0,0 +1,35 @@
+package subscription
+
+import (
+	"context"
+	"fmt"
+
+	"go.temporal.io/sdk/activity"
+)
+
+type Activities struct{}
+
+func (a *Activities) SendWelcomeEmail(ctx context.Context, subscription SubscriptionInfo) error {
+	logger := activity.GetLogger(ctx)
+	logger.Info("Sending welcome email to " + subscription.Customer.Email)
```

Comment:
```suggestion
	logger.Info("Sending welcome email", "CustomerEmail", subscription.Customer.Email)
```

It's often better to have variable data as KV tags in the logger. But this is a minor thing, can ignore.